examples/app.py

```diff
@@ -103,6 +106,11 @@
 InvenioRecords(app)
 InvenioSearch(app)
 InvenioSearchUI(app)
+InvenioPIDStore(app)
+InvenioRecordsREST(app)
```

Comment:
The example works because you register REST endpoint on "default" application object.